### PR TITLE
🌱 Run `Dangerous-Workflow` in release tests

### PR DESCRIPTION
**Pull request by azeemshaikh38**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Start running the `Dangerous-Workflow` check in the release tests to build confidence before we enable it in production.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

### CODE REVIEW

**Review by asraa — APPROVED** [review]

File: cron/config/config.yaml [hunk 1]
```diff
@@ -22,6 +22,10 @@ completion-threshold: 0.99
 shard-size: 10
 webhook-url: 
 cii-data-bucket-url: gs://ossf-scorecard-cii-data
+# TODO: Temporarily remove SAST and CI-Tests which require lot of GitHub API tokens.
```
Comment (laurentsimon) [inline cron/config/config.yaml:25]: note: ISE wants to use CI-Tests it in their dashboard

**Review by laurentsimon — CHANGES_REQUESTED** [review]
See comment on ISE above.

File: cron/config/config.yaml (hunk 1 above)
Comment (azeemshaikh38) [reply cron/config/config.yaml:25]: To be clear, `CI-Tests` have been disabled for a quite a while now. This PR is not disabling any new checks. 

If we want to re-enable `CI-Tests`, need to look into making the check more performant before adding in back in. Don't have the cycles to do that this quarter. Can you create an issue and maybe we can attempt this in Q1?

Comment (laurentsimon) [reply cron/config/config.yaml:25]: Good to know! Which other checks are currently disabled?

**Review by laurentsimon — APPROVED** [review]
CI-Tests was already disabled. So approving. Can we add this info to the README under cron section?

File: cron/config/config.yaml (hunk 1 above)
Comment (azeemshaikh38) [reply cron/config/config.yaml:25]: SAST, CI-Test and Contributors. We had even updated the doc with this info, IIRC. Can't seem to find it now.

**Pull request merged**